doc/QUICK_START.org

```diff
@@ -16,11 +16,25 @@
 
 * Configuration layers
 Spacemacs divides its configuration into self-contained units called
-=configuration layers=. These layers are stacked on top of each other
-to achieve a custom configuration.
+=configuration layers=. These layers are stacked on top of each other to achieve
+a custom configuration.
 
 By default Spacemacs uses a dotfile called =~/.spacemacs= to control which
-layers to load. Within this file you can also configure certain features.
+layers to load. Within this file you can also configure certain features. This
+design makes it extremely easy to turn Spacemacs into a fully integrated
+development environment.
+
+For example to get python support, simply add =python= to the list of
+=dotspacemacs-configuration-layers= in your =~/.spacemacs=. Find the
+=~/.spacemacs= file by pressing =SPC f e d=, and after adding the layer reload
+it with =SPC f e R=. Now open a =.py= file (=SPC f f=) to find the python
+environment has been fully configured. Some small extra configurations might
+make the environment even more powerful, which is very well described in the
+[[https://develop.spacemacs.org/layers/+lang/python/README.html][layer's documentaion]].
+
+A list of pre-configured layers is available [[https://develop.spacemacs.org/layers/LAYERS.html][here]]. If you still would like to
+configure things not covered by any of the layers then it it easy to build a
+personal layer.
```

Comment:
You can also mention the in-app key binding `SPC h l` or `SPC h SPC` (which combines several sources into one).

Reply:
Good point